### PR TITLE
Workload Cluster Azure Release 14.0.1

### PR DESCRIPTION
**Pull request by T-Kukawka**
Issue: https://github.com/giantswarm/giantswarm/issues/16181

### CODE REVIEW

File: azure/v14.0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# :zap: Giant Swarm Release v14.0.1 for Azure :zap:
+
+This releases increases the Azure Events Termination timeout from 5 to 15 minutes for better upgrade experience while workloads are moved to new nodes
```
Comment (whites11) [inline azure/v14.0.1/README.md:3]:
```suggestion
This releases increases the Azure Events Termination timeout from 5 to 15 minutes for better upgrade experience while workloads are moved to new nodes.
```

Comment (whites11) [reply azure/v14.0.1/README.md:3]: Does it make sense to mention the fact we updated the azure-scheduled-events app to make upgrade smoother?

**Review by fiunchinho — APPROVED** [review]

File: azure/v14.0.1/README.md (hunk 1 above)
Comment (T-Kukawka) [reply azure/v14.0.1/README.md]: It is sort of in the applied changes

**Review by whites11 — APPROVED** [review]

**Pull request merged**